Source/externs.m

```diff
@@ -755,6 +768,7 @@
 NSString *NSImageNameColorPanel = @"NSColorPanel";
 NSString *NSImageNameCaution = @"NSCaution";
 
+NSString *_NXSmartPaste = @"NeXT smart paste pasteboard type";
```

Comment:
Could you please move this constant closer to the pasteboard types? (Around line 300) Otherwise it is really hard to understand what it is used for.